clients/python/sysadminctl

```diff
@@ -54,7 +56,7 @@ def format_kvs(kvs):
     for kv in kvs:
         if kv.value:
             value = UnpackFromProto(kv.value)
-            if isinstance(value, str) or isinstance(value, unicode):
+            if isinstance(value, str) or isinstance(value, str):
```

Comment:
did you update this to python 3? If so, this `unicode` check my not be necessary, but this is definitely redundant  if so

Reply:
Yup, that was generated by `2to3` with the hope of supporting both for free, but I obviously didn't look at the output carefully. The same problem exists in `Client._convertTypes` which calls `unicode()`. I'll revert it